action-server/covidflow/utils/persistence.py

```diff
@@ -1,89 +1,115 @@
-import logging
 from typing import Any, Dict, Text
 
+from structlog import get_logger
+
 from covidflow.db.assessment import Assessment
 from covidflow.db.base import session_factory
 from covidflow.db.reminder import Reminder
 
 from .hashids_util import decode_reminder_id, encode_reminder_id
 from .phone_number_validation import is_test_phone_number
 
-logger = logging.getLogger(__name__)
+logger = get_logger(__name__)
 
+PHONE_NUMBER_SLOT = "phone_number"
 METADATA_SLOT = "metadata"
 REMINDER_ID_METADATA_PROPERTY = "reminder_id"
 
 
-def get_reminder_id(slot_values: Dict[Text, Any]) -> int:
-    metadata = slot_values.get(METADATA_SLOT, {})
+def _get_reminder_id(slot_values: Dict[Text, Any]) -> int:
```

Comment:
You have private functions defined at the top and the bottom of the file. We are not coherent between files, but in one file they should all be at the same place

Reply:
Moved everything to the bottom